tools/xtce_generation/telemetry_generator.py

```diff
@@ -157,7 +168,9 @@ def create_parameter_types(self, parameter_type_set, unique_lengths):
         for parameter_type_ref_name, size in unique_lengths.items():
             if "UINT" in parameter_type_ref_name:
                 parameter_type = Et.SubElement(
-                    parameter_type_set, "xtce:IntegerParameterType"
+                    parameter_type_set,  # type: ignore[arg-type]
+                    # ToDo Change, expected Element type for argument 1
```

Comment:
```suggestion
                    parameter_type_set,
```

Should be able to remove these with the fixes described above.